lib/clients/statsd.js

```diff
@@ -0,0 +1,78 @@
+var BaseClient = require('./base');
+var util = require('util');
+var async = require('async');
+var types = require('../types');
+var _ = require('lodash');
+var dgram = require('dgram');
+
+var STATS_TYPES = {};
+STATS_TYPES[types.C] = 'c';
+STATS_TYPES[types.G] = 'g';
+STATS_TYPES[types.T] = 'ms';
+
+var KEY_DELIMITER = "-";
+
+function sanatizeInput(input) {
+  //Replace characters that may cause problems here (:| are used by statsd, and cannot be included in the name)
+  return (input + '').replace(/[\s:|=<.]+/g, '');
+}
+
+/**
+ * A client that can send metrics data to a statsd backend
+ * @param {Object} opts options about the statsd backend
+ * @param {String} opts.host the host of the statsd server. Default is 127.0.0.1.
+ * @param {Number} opts.port the port of the statsd server. Default is 8125.
+ */
+function StatsdClient(opts) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Rename this 'StatsdClient' function to match the regular expression ^[a-z][a-zA-Z0-9]*$ [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS100)